depends/packages/native_zstd.mk

```diff
@@ -0,0 +1,34 @@
+package=native_zstd
```

Comment:
There needs to be a copyright entry for this new dependency in `contrib/debian/copyright`. (It's dual BSD or GPLv2.)

Reply:
This can be resolved.